torchvision/io/image.py

```diff
@@ -68,7 +69,9 @@ def write_file(filename: str, data: torch.Tensor) -> None:
 
 
 def decode_png(
-    input: torch.Tensor, mode: ImageReadMode = ImageReadMode.UNCHANGED, apply_exif_orientation: bool = False
+    input: torch.Tensor,
+    mode: ImageReadMode = ImageReadMode.UNCHANGED,
+    apply_exif_orientation: bool = False,
```

Comment:
Nit: best to avoid that kind of unrelated formatting changes (I assume those come from the internal linter?) as they distract from the review and they also add noise to `git blame`.

You can keep them here as those aren't too disruptive, just letting you know for future PRs.

Reply:
Got it - yea, I think that was the linter.